setup.py

```diff
@@ -28,7 +29,7 @@
 
 setup(
     name="metar",
-    version="1.5.0",
+    version=__version__,
     author="Tom Pollard",
     author_email="pollard@alum.mit.edu",
     url="http://github.com/tomp/python-metar",
```

Comment:
while we're in there, we should update some of this meta data as well

Reply:
Was planning to do a bunch of that with the formal release PR.